build.sbt

```diff
@@ -22,15 +22,13 @@ lazy val buildSettings = Seq(
   publishMavenStyle := true,
   publishArtifact in Test := false,
   pomIncludeRepository := { _ => false },
-  licenses := Seq("Apache 2.0" -> new URL("http://www.apache.org/licenses/LICENSE-2.0.txt")),
+  licenses += ("Apache-2.0", url("http://www.apache.org/licenses/LICENSE-2.0.html")),
   homepage := Some(url("https://github.com/allenai/datastore")),
+  apiURL := Some(url("https://allenai.github.io/datastore/")),
```

Comment:
Followed the `common` project example. That's supposed to be the API URL. The `common` one also 404's from the browser.